src/extension/ncwms/src/main/java/au/org/emii/geoserver/wms/Ncwms.java

```diff
@@ -122,9 +124,22 @@ private void proxyWmsRequest(HttpServletRequest request, HttpServletResponse res
 
             URL wmsUrl = new URL(wmsUrlStr + "?" + queryString);
 
-            IOUtils.copy(wmsUrl.openConnection().getInputStream(), response.getOutputStream());
+            HttpURLConnection connection = (HttpURLConnection)wmsUrl.openConnection();
```

Comment:
Would you know? In the catch you would not get a reference to the connection, only the exception?